src/timeline/InputBar.cpp

```diff
@@ -846,6 +846,9 @@ InputBar::command(const QString &command, QString args)
                                 cache::getMembers(this->room->roomId().toStdString(), 0, -1));
     } else if (command == QLatin1String("converttoroom")) {
         utils::removeDirectFromRoom(this->room->roomId());
+    } else {
+        if (true)
```

Comment:
?

Reply:
LOL, I forgot to delete some temporary scaffolding, it appears